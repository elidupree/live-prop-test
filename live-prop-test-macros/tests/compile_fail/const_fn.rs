use live_prop_test::live_prop_test;

#[live_prop_test(postcondition = "result == 5")]
const fn five() -> i32 {
  5
}

fn main() {}
