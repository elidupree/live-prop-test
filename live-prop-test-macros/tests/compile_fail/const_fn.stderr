error: live-prop-test doesn't support testing const fn items
 --> $DIR/const_fn.rs:4:1
  |
4 | const fn five() -> i32 {
  | ^^^^^
